web/themes/custom/imaginelex/_patterns/molecules/hero-with-title-and-body.twig

```diff
@@ -0,0 +1,7 @@
+<div class="hero hero--container">
```

Comment:
wouldn't this be the same as saying `include @components/molecules/hero-with-title.twig`? Let's change that and then you can merge this.